packages/orchestrator-ui-components/src/components/WFOSteps/getStyles.ts

```diff
@@ -0,0 +1,81 @@
+import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
+import { css } from '@emotion/react';
+// import { makeHighContrastColor } from '@elastic/eui';
+
+export const getStyles = (theme: EuiThemeComputed) => {
```

Comment:
Whole file: try to use data from the theme to populate the numeric values. You could use the theme.base value to multiply from there. There is also a multiplyBase() helper function in our theme, unless you pass it in here as a parameter

Reply:
For the stepState icons the values are very tightly coupled. Changing the width will require a another value for the transform to get the icon to position correctly on top of the other and that might not be in de same proportions so I do want to make those fixed tbh.